adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/InMemoryGtfsDataRepository.java

```diff
@@ -0,0 +1,33 @@
+package org.mobilitydata.gtfsvalidator.db;
+
+import org.mobilitydata.gtfsvalidator.domain.entity.gtfsentity.Agency;
+import org.mobilitydata.gtfsvalidator.usecase.port.GtfsDataRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class InMemoryGtfsDataRepository implements GtfsDataRepository {
+
+    public List<Agency> getAgencyList() {
```

Comment:
Nomenclature: replace by getAgencyCollection without touching the return type so that the method name can cover all future possibilities for the collection type.